pkg/controller/job/job_controller.go

```diff
@@ -140,6 +140,7 @@ type syncJobCtx struct {
 	uncounted                       *uncountedTerminatedPods
 	podsWithDelayedDeletionPerIndex map[int]*v1.Pod
 	terminating                     *int32
+	ready                           *int32
```

Comment:
I think it does not need to be a pointer, since the feature gate is already dropped?